### PR TITLE
Add kube-dns service account

### PR DESCRIPTION
**Pull request by at1012**

### CODE REVIEW

File: salt/addons/init.sls [hunk 1]
```diff
@@ -15,6 +15,11 @@ include:
     - template:    jinja
 
 {% if salt['pillar.get']('addons:dns', 'false').lower() == 'true' %}
+/etc/kubernetes/addons/kubedns-sa.yaml:
+  file.managed:
+    - source:     salt://addons/addons/kubedns-sa.yaml.jinja
+    - template:   jinja
```
Comment (inercia) [inline salt/addons/init.sls]: Nothing is replaced in this file, so no Jinja templating is needed,

File: salt/addons/deploy_addons.sh [hunk 2]
```diff
@@ -26,9 +26,19 @@ fi
 
 echo "Deploying DNS on Kubernetes"
 
+KUBEDNS_SA=`eval "$KUBECTL_PATH $NS_ARGS $SRV_ARGS get sa | grep kube-dns | cat"`
```
Comment (inercia) [inline salt/addons/deploy_addons.sh]: I think we should try to move all this stuff from the `deploy_addons.sh` script to some state file, and just invoke `kubectl` with `cmd.run`.

File: salt/addons/init.sls (hunk 1 above)
Comment (robdaemon) [inline salt/addons/init.sls]: It's harmless.

File: salt/addons/deploy_addons.sh (hunk 2 above)
Comment (robdaemon) [inline salt/addons/deploy_addons.sh]: That's outside of the scope of this change. A replacement of this script is in progress.

**Pull request merged**